app/views/team_members/_table.html.erb

```diff
@@ -4,20 +4,19 @@
       <tr class="govuk-table__row">
         <td class="govuk-table__cell "><%= member.email %></td>
         <td class="govuk-table__cell ">
-	  <% if !member.invitation_sent_at.nil? && !member.invitation_accepted? %>
-            invitation pending
-	  <% end %>
-	</td>
+          <% if !member.invitation_sent_at.nil? && !member.invitation_accepted? %>
```

Comment:
`!smthng.nil?` is better expressed as `smthng.present?`or just `if member.invitation_sent_at`. Although in this case I would almost be tempted to drop a method on the `User` model:
```
  def invitation_pending?
    invitation_sent_at && !invitation_accepted?
  end
```
Then you just use `if member.invitation_pending?`

Reply:
I actually just changed the tabbing of this bit! 

I'm happy to sort this out whilst I'm here, if it's not making the PR too big.

Reply:
Fixed in https://github.com/alphagov/govwifi-admin/pull/106/commits/ce4e5cbcce83542aa7c33a86fec12b705a0b636d